lib/s3middleware/objectLegalHold.js

```diff
@@ -0,0 +1,113 @@
+const { parseString } = require('xml2js');
+const errors = require('../errors');
+
+/*
+    Format of the xml request:
+    <LegalHold>
+        <Status>ON|OFF</Status>
+    </LegalHold>
+*/
+
+/**
+ * @param {object[]} status - legal hold status parsed from xml to be validated
```

Comment:
is this an object or a string?

Reply:
it's a string inside array (object supposed to be string)